models/Group.model.js

```diff
@@ -0,0 +1,33 @@
+const { Schema, model } = require("mongoose");
+
+const groupSchema = new Schema(
+  {
+    groupName: {
+      type: String,
+      require: true,
+      unique: true
+    },
+    description: {
+      type: String,
+    },
+    members: [
+      {
+        type: Schema.Types.ObjectId,
+        ref: "User"
+      },
+    ],
+    movies: [
+      {
+        type: Schema.Types.ObjectId.apply,
```

Comment:
will cause an error: 

```suggestion
type: Schema.Types.ObjectId
```